source/agora/consensus/protocol/Nominator.d

```diff
@@ -695,11 +695,11 @@ extern(D):
             }
         } catch (Exception ex)
         {
-            assert(0, format!"[%s:%s] combineCandidates: slot %u. Exception: %s"(
+            assert(0, format!"combineCandidates: slot %u. Exception: %s"(
                 __FILE__, __LINE__, slot_idx, ex.to!string));
```

Comment:
Should you also remove `__FILE__` and `__LINE__`?

Reply:
Updated

Reply:
@hewison-chris, Did you find the way how to go to the file when clicking the line of logs?

Reply:
No I couldn't find the option or an extension for this behaviour in VSCode. However quite often the stack trace has the clickable link to the file also.